make fails

Make is failing for me with the following:

jnankin@bruno:~/golang/src/github.com/NYTimes/mock-ec2-metadata$ make
/home/jnankin/golang/bin/gox -output "bin/{{.Dir}}_0.3.1_{{.OS}}_{{.Arch}}" -os="linux" -os="darwin" -arch="386" -arch="amd64" ./
Number of parallel builds: 3

-->    darwin/amd64: github.com/NYTimes/mock-ec2-metadata
-->     linux/amd64: github.com/NYTimes/mock-ec2-metadata
-->       linux/386: github.com/NYTimes/mock-ec2-metadata
-->      darwin/386: github.com/NYTimes/mock-ec2-metadata

4 errors occurred:
--> linux/amd64 error: exit status 2
Stderr: # github.com/NYTimes/gizmo/server
../gizmo/server/rpc_server.go:227:12: undefined: metadata.FromContext

--> darwin/amd64 error: exit status 2
Stderr: # github.com/NYTimes/gizmo/server
../gizmo/server/rpc_server.go:227:12: undefined: metadata.FromContext

--> linux/386 error: exit status 2
Stderr: # github.com/NYTimes/gizmo/server
../gizmo/server/rpc_server.go:227:12: undefined: metadata.FromContext

--> darwin/386 error: exit status 2
Stderr: # github.com/NYTimes/gizmo/server
../gizmo/server/rpc_server.go:227:12: undefined: metadata.FromContext

Makefile:31: recipe for target 'build' failed
make: *** [build] Error 1

I have the same problem and got further in the following way:
```
$ go get github.com/Sirupsen/logrus
```

(missing from `Godeps`)

and then make this change:

```
$ git diff
diff --git a/main.go b/main.go
index 2434b73..e6da228 100644
--- a/main.go
+++ b/main.go
@@ -5,7 +5,8 @@ import (
 
        "github.com/NYTimes/gizmo/config"
        "github.com/NYTimes/gizmo/server"
-       "github.com/NYTimes/mock-ec2-metadata/service"
+       "./service"
 )
```

With that said, I'm getting an error which I presume is to do with golang version (I'm on 1.9.4, Ubuntu):

```
../../vendor/gostuff/src/github.com/NYTimes/gizmo/server/rpc_server.go:227:12: undefined: metadata.FromContext
``

Quick and dirty try, still failing on some deps.
Could anyone move this forward, maybe?

```docker
FROM library/golang

ARG DST_DIR=/go/src/github.com/NYTimes/mock-ec2-metadata

RUN go get github.com/Sirupsen/logrus

RUN mkdir -p $DST_DIR
COPY . $DST_DIR

RUN ls -la $DST_DIR
RUN cd $DST_DIR \
    && make deps 

RUN cd $DST_DIR \
    && make 
```

Then run:
```sh
docker build . \
    -t mock-ec2-metadata:build  \
    --file Dockerfile
```

Failing:
```
/go/bin/gox -output "bin/{{.Dir}}_0.3.1_{{.OS}}_{{.Arch}}" -os="linux" -os="darwin" -arch="386" -arch="amd64" ./
Number of parallel builds: 3

-->    darwin/amd64: github.com/NYTimes/mock-ec2-metadata
-->     linux/amd64: github.com/NYTimes/mock-ec2-metadata
-->      darwin/386: github.com/NYTimes/mock-ec2-metadata
-->       linux/386: github.com/NYTimes/mock-ec2-metadata

4 errors occurred:
--> darwin/386 error: exit status 2
Stderr: # github.com/NYTimes/gizmo/server
../gizmo/server/rpc_server.go:227:12: undefined: metadata.FromContext

--> linux/amd64 error: exit status 2
Stderr: # github.com/NYTimes/gizmo/server
../gizmo/server/rpc_server.go:227:12: undefined: metadata.FromContext

--> darwin/amd64 error: exit status 2
Stderr: # github.com/NYTimes/gizmo/server
../gizmo/server/rpc_server.go:227:12: undefined: metadata.FromContext

--> linux/386 error: exit status 2
Stderr: # github.com/NYTimes/gizmo/server
../gizmo/server/rpc_server.go:227:12: undefined: metadata.FromContext
```

--- a/.gitignore
+++ b/.gitignore
@@ -1,1 +1,3 @@
 bin
+mock-metadata-access.log
+mock-metadata-app.log

--- a/Godeps
+++ b/Godeps
@@ -1,1 +1,1 @@
-github.com/NYTimes/gizmo 12c3a2a0a5dcc7cdc6f3d13b1ceae135872f786a
+github.com/NYTimes/gizmo bb17238799b62b40a3bcb83a072d980562071436

--- a/Makefile
+++ b/Makefile
@@ -26,6 +26,7 @@ deps:
 	go get github.com/mtibben/gogpm
 	go get github.com/tcnksm/ghr
 	go get github.com/mitchellh/gox
+	go get github.com/Sirupsen/logrus
 	$(GOPATH)/bin/gogpm install
 
 build:

